### DISCUSSION TITLE
Latest Z2M update is broken

### DISCUSSION THREAD

**Discussion by davek00997744** [Koenkk/zigbee2mqtt#12931] [General]
Seems the Tuya/Zemismart cover motor zigbee2mqtt feature is now broken?

The blinds are not reporting position/status anymore - i manually opened one and its position is the same as when it was closed. Node red shows current position as 100 if it’s open or not.

If i go into the MQTT section of HA, each blind has control buttons and they function correctly - open will open and close will close them, however the “current position” is not updating so it seems the latest Zigbee/MQTT update has broken position reporting?

Is there a way i can fix this, sort of important as we have a few of these and all are now useless??

Notes...
The “current position” status does not change
In NodeRed, i used to use “set position” = 0 to close, now its 100, so it has reversed logic
In Zigbee2MQTT devices i can see the “Settings:specific” option but it makes no difference.

On my Home Assistant Pi, I updated the Core, the HA system and Zigbee2MQTT add-on, I run updates every month and always check the breaking changes list but saw nothing to make me think it would upset things. 99% of the times it works but this time it bit me :frowning:

In the zigbee2MQTT log i now see this…

Zigbee2MQTT:error 2022-06-25 06:36:01: Request ‘zigbee2mqtt/bridge/request/device/configure_reporting’ failed with error: ‘ConfigureReporting 0x5c0272fffec20d2d/1 genOta([{“attribute”:"",“minimumReportInterval”:60,“maximumReportInterval”:3600,“reportableChange”:0}], {“sendWhen”:“immediate”,“timeout”:10000,“disableResponse”:false,“disableRecovery”:false,“disableDefaultResponse”:true,“direction”:0,“srcEndpoint”:null,“reservedBits”:0,“manufacturerCode”:null,“transactionSequenceNumber”:null,“writeUndiv”:false}) failed (Don’t know value type for ‘undefined’)’
Zigbee2MQTT:error 2022-06-25 06:36:20: Request ‘zigbee2mqtt/bridge/request/device/configure_reporting’ failed with error: ‘ConfigureReporting 0x5c0272fffec20d2d/1 genOta([{“attribute”:"",“minimumReportInterval”:60,“maximumReportInterval”:3600,“reportableChange”:0}], {“sendWhen”:“immediate”,“timeout”:10000,“disableResponse”:false,“disableRecovery”:false,“disableDefaultResponse”:true,“direction”:0,“srcEndpoint”:null,“reservedBits”:0,“manufacturerCode”:null,“transactionSequenceNumber”:null,“writeUndiv”:false}) failed (Don’t know value type for ‘undefined’)’

**Comment by Koenkk** [comment]
See https://github.com/Koenkk/zigbee2mqtt/issues/12719

**Reply by davek00997744** [reply]
Thanks, what do i need to do - that thread does not make too much sense to a noob like me :)

**Reply by Koenkk** [reply]
Wait till the next release (on 1 Jul)

**Reply by davek00997744** [reply]
Great, thanks for that
